### DISCUSSION TITLE
Databases like in Notion

### DISCUSSION THREAD

**Discussion by davidkobielski** [outline/outline#7709] [Ideas]
Hello All!

First of all, I want to commend you on the amazing work you've done with Outline so far! The app is incredibly useful and has become an essential tool for many teams, including mine.

I was wondering if you've considered taking Outline in a slightly different direction by introducing database functionality, similar to what Notion offers? Having the ability to create, manage, and relate databases would add an entirely new level of functionality to Outline, making it an even more powerful tool for knowledge management.

The current state of Outline is great, but adding databases would really elevate the experience and open up new possibilities for organizing information. It would allow users to not only document and collaborate but also to build dynamic data-driven workflows directly within the platform.

Thank you for your time, and I look forward to hearing your thoughts!

Best regards,
Dawid

**Comment by wforte88** [comment]
This would be perfect for establishing relationships between articles and categorizing them better.

**Comment by ghost** [comment]
Hey everyone! I’ve started a new discussion on adding **structured data (databases)** to Outline and referenced this thread among other related ideas.

The new post is quite visual and includes a current ecosystem overview, potential features, and mockups. I’d love your feedback:
⭐ #8476
